### PR TITLE
CC-1174: Config new parameters for UoCbyCollectionType report

### PR DESCRIPTION
**Pull request by lkvoong**
New report parameters:

- StaffName
- VisitStartDate
- VisitEndDate

### CODE REVIEW

File: src/plugins/invocables/report/pahmaUoCbyCollectionType/fields.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        view: {
+          type: CompoundInput,
+        },
+      },
+      StaffName: {
+        [config]: {
+          messages: defineMessages({
+            name: {
+              id: 'field.pahmaUoCbyCollectionType.StaffName.name',
+              defaultMessage: 'Staff Name',
```
Comment (cesarvh) [inline src/plugins/invocables/report/pahmaUoCbyCollectionType/fields.js]: Staff name

File: src/plugins/invocables/report/pahmaUoCbyCollectionType/fields.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+            },
+          },
+        },
+      },
+      VisitStartDate: {
+        [config]: {
+          messages: defineMessages({
+            name: {
+              id: 'field.pahmaUoCbyCollectionType.VisitStartDate.name',
+              defaultMessage: 'Visit Start Date',
```
Comment (cesarvh) [inline src/plugins/invocables/report/pahmaUoCbyCollectionType/fields.js]: Visit start date

File: src/plugins/invocables/report/pahmaUoCbyCollectionType/fields.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+            type: DateInput,
+          },
+        },
+      },
+      VisitEndDate: {
+        [config]: {
+          messages: defineMessages({
+            name: {
+              id: 'field.pahmaUoCbyCollectionType.VisitEndDate.name',
+              defaultMessage: 'Visit End Date',
```
Comment (cesarvh) [inline src/plugins/invocables/report/pahmaUoCbyCollectionType/fields.js]: Visit end date

**Review by cesarvh — CHANGES_REQUESTED** [review]
Hey Lam, same as PR #46

File: src/plugins/invocables/report/pahmaUoCbyCollectionType/fields.js (hunk 1 above)
Comment (lkvoong) [reply src/plugins/invocables/report/pahmaUoCbyCollectionType/fields.js]: Fixed. Thank you!

File: src/plugins/invocables/report/pahmaUoCbyCollectionType/fields.js (hunk 3 above)
Comment (lkvoong) [reply src/plugins/invocables/report/pahmaUoCbyCollectionType/fields.js]: Fixed. Thank you!

File: src/plugins/invocables/report/pahmaUoCbyCollectionType/fields.js (hunk 2 above)
Comment (lkvoong) [reply src/plugins/invocables/report/pahmaUoCbyCollectionType/fields.js]: Fixed. Thank you!

**Pull request merged**